### PR TITLE
CRUD Generator: Correctly support `type: "text"` fields in filter and sort

### PR DESCRIPTION
**Pull request by nsams**
fixes test added in https://github.com/vivid-planet/comet/pull/1491 (althouth differently)

### CODE REVIEW

**Comment by johnnyomair** [comment]
We also need this for input. Do you want to add this in this PR?

**Comment by nsams** [comment]
input is already supported https://github.com/vivid-planet/comet/blob/main/packages/api/cms-api/src/generator/generate-crud-input.ts#L87

**Comment by johnnyomair** [comment]
> input is already supported https://github.com/vivid-planet/comet/blob/main/packages/api/cms-api/src/generator/generate-crud-input.ts#L87

But it generated `field: text;`. Fixed in #1497

File: .changeset/lovely-rats-rest.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-api": patch
+---
+
+CRUD Generator: correctly support type="text" fields in filter and sort
```
Comment (johnnyomair) [inline .changeset/lovely-rats-rest.md:5]:
```suggestion
CRUD Generator: Correctly support `type: "text"` fields in filter and sort
```

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**